csvs/bitbank-spot-trade-rev2.json

```
@@ -0,0 +1,271 @@
+{
+  "id": "bitbank-spot-trade-rev2",
```

```suggestion
  "id": "bitbank-trade",
```

現物だけじゃなくなってるので、 `spot` は ID から消したほうが良いかも